core/types/transaction.go

```diff
@@ -95,6 +95,8 @@ type Transaction interface {
 	SetSender(libcommon.Address)
 	IsContractDeploy() bool
 	IsDepositTx() bool
+	// Legacy deposit tx
```

Comment:
Nit, but I'm not sure this comment adds any value, and breaks Go conventions by not starting with `IsLegacyDepositTx`.

Reply:
Removed.